Gems/ROS2/Code/Source/Frame/ROS2FrameComponent.cpp

```diff
@@ -114,7 +137,7 @@ namespace ROS2
 
     bool ROS2FrameComponent::IsDynamic() const
     {
-        return IsTopLevel();
+        return m_isDynamic;
```

Comment:
Nice. If we consider 
Dynamic = not having a fixed joint
then the top level logic is also retained.